browser/net/sources.gni

```diff
@@ -77,13 +76,13 @@ brave_browser_net_deps = [
   "//brave/components/constants:brave_service_key_helper",
   "//brave/components/decentralized_dns/content",
   "//brave/components/decentralized_dns/core",
-  "//brave/components/ipfs/buildflags",
   "//brave/components/localhost_permission",
   "//brave/components/query_filter",
   "//brave/components/update_client:buildflags",
   "//brave/extensions:common",
   "//chrome/browser:browser_process",
   "//chrome/browser/profiles",
+  "//components/base32:base32",
```

Comment:
what is it for? also you can omit trailing `:base32`

Reply:
removed it